dpar/src/features/lookup.rs

```diff
@@ -91,6 +98,10 @@ impl Lookup for MutableLookupTable {
         Some(numberer.add(feature.to_owned()))
     }
 
+    fn lookup_values(&self, values: &mut Vec<String>) {
```

Comment:
Same here, though the `borrow` may make things a bit ugly.

Reply:
Yes, that is why I went for the detour of modifying a vec that's passed as an argument

Reply:
The `borrow` is annoying, but in these cases you can return a `Cow` value.

https://doc.rust-lang.org/std/borrow/enum.Cow.html

Return a borrowed copy when you can, otherwise return an owned value.